lms/djangoapps/grades/tasks.py

```diff
@@ -91,9 +90,6 @@ def _recalculate_subsection_grade(self, **kwargs):
     """
     try:
         course_key = CourseLocator.from_string(kwargs['course_id'])
-        if not PersistentGradesEnabledFlag.feature_enabled(course_key):
-            return
```

Comment:
Well, it means it is now on for edge (async calculation but not persistent?)
And we for example run smaller celery installations in some openedx installs that we manage because they don't use many async tasks.